integrations/webchat/examples/custom-element/client/javascript-animation/index.html

```diff
@@ -81,6 +85,8 @@
           // Move the main window to the off-screen position and then un-hide it.
           instance.elements.getMainWindow().addClassName('StartOpenAnimation');
           instance.elements.getMainWindow().removeClassName('HideWebChat');
+          // Make the custom element visible.
+          customElement.classList.remove('WebChatContainer--hidden');
```

Comment:
I don't think this should be here. One of the points of this tutorial is to demonstrate how to animate the opening and closing of web chat and still use our built-in launcher. If you hide the custom element, you can't do that. And in fact, because of this, this example doesn't actually work. If you open this html file as-is, the custom element is hidden and the launcher is invisible.

Reply:
If I remove the changes to the hide the custom element, the entrance animation doesn't work. Web chat just instantly appears. The exit animation does however work.